src/main/java/frc/robot/controls/Driver.java

```diff
@@ -13,12 +13,16 @@
 
 package frc.robot.controls;
 
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.*;
+import frc.robot.Constants;
+import frc.robot.commands.*;
 import frc.robot.Robot;
 
 public class Driver
 {
     // JOYSTICKS / CONTROLLERS //
-
+    private Joystick dualshock;
```

Comment:
Is dualshock a joystick or an Xbox/playstation controller?  You may want to ask @droid-101 what kind of controller he will be driving with.  If dualshock is used by the operator then you may want to move this code to `Operator.java`.

Reply:
The dualshock is a generic playstation controller. I'm not yet sure what kind of controller @droid-101 wants to use, but I can always move the code over in a later pull request.

Reply:
Sounds good to me!

Reply:
Ok, I heard from @droid-101 that he wants 2 joysticks for the driver and a controller for the operator. Should I change that here or do that in a different pull request?